csrankings-j.csv

```diff
@@ -2071,10 +2072,12 @@ Jugal K. Kalita,UCCS,http://www.cs.uccs.edu/~kalita,NOSCHOLARPAGE
 Jugal Kalita,UCCS,http://www.cs.uccs.edu/~kalita,NOSCHOLARPAGE
 Jugal Kumar Kalita,UCCS,http://www.cs.uccs.edu/~kalita,NOSCHOLARPAGE
 Jugurta Lisboa Filho,Universidade Federal de Viçosa,http://www.dpi.ufv.br/~jugurta,gQN4nmkAAAAJ
+Juhan Nam,KAIST,https://mac.kaist.ac.kr,eG2NHUYAAAAJ

```

Comment:
In "Graduate School of Culture Technology", not clearly CS. Please confirm (read over the eligibility requirements).

Reply:
Same, found it.